less/variables.less

```diff
@@ -11,4 +11,35 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+
+@mobile: ~"(max-width: 500px)";
+
+@media @mobile{
+  
```

Comment:
Mobile styles can be placed in the file they pertain to. So mobile header styles can be in the navigation file etc.